shared/common-adapters/avatar.render.native.js

```diff
@@ -207,17 +178,48 @@ class AvatarRender extends React.PureComponent<Props, State> {
 const sizes = [176, 112, 80, 64, 48, 40, 32, 24, 16, 12]
 
 const iconStyles = sizes.reduce((map, size) => {
-  map[String(size)] = {height: size, width: size}
+  map[`icon:${size}`] = {height: size, width: size}
   return map
 }, {})
 
 const boxStyles = sizes.reduce((map, size) => {
-  map[String(size)] = {height: size, position: 'relative', width: size}
+  map[`box:${size}`] = {height: size, position: 'relative', width: size}
+  return map
+}, {})
+
+const imageStyles = sizes.reduce((map, size) => {
+  map[`image:${size}`] = {
+    bottom: 0,
+    height: size,
+    left: 0,
+    position: 'absolute',
+    right: 0,
+    top: 0,
+    width: size,
+  }
   return map
 }, {})
 
-const iconStyle = (size, style) =>
-  style ? {...iconStyles[String(size)], ...style} : iconStyles[String(size)]
-const boxStyle = (style, size) => (style ? {...boxStyles[String(size)], ...style} : boxStyles[String(size)])
+const styles = styleSheetCreate({
```

Comment:
using the stylesheet create to optimize sending styles over the bridge

Reply:
maybe have this as a comment?

Reply:
but this will be a policy for everything. i think adding a comment in this one component isn't what we want